File: yangxj96-serve/yangxj96-serve-system/src/main/java/io/github/yangxj96/server/system/Application.java
package io.github.yangxj96.server.system;

import io.github.yangxj96.starter.db.annotation.EnableDynamicDatasource;
import org.mybatis.spring.annotation.MapperScan;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

/**
 * 系统服务的主启动类
 */
@MapperScan("io.github.yangxj96.server.system.mapper")
@EnableDynamicDatasource
@SpringBootApplication
public class Application {

    public static void main(String[] args) {
        SpringApplication.run(Application.class, args);
    }

}

File: yangxj96-serve/yangxj96-serve-demo/src/main/java/io/github/yangxj96/server/demo/Application.kt
/*
 *  Copyright (c) 2021 - 2023
 *  作者：杨新杰(Jack Young)
 *  邮箱：yangxj96@gmail.com
 *  日期：2023-07-25 15:31:55
 *  Copyright (c) 2021 - 2023
 */
package io.github.yangxj96.server.demo

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

/**
 * 示例模块
 */
@SpringBootApplication
class Application

fun main(args: Array<String>) {
    runApplication<Application>(*args)
}